ariadne/wsgi.py

```diff
@@ -2,7 +2,7 @@
 from inspect import isawaitable
 from typing import Any, Callable, Dict, List, Optional, Type, Union, cast
 from urllib.parse import parse_qsl
-
+from http import HTTPStatus
```

Comment:
Sort imports and separate stdlib imports from 3rd party imports with a blank line.